zigbee2mqtt-edge/config.json

```diff
@@ -1,6 +1,7 @@
 {
   "name": "Zigbee2MQTT Edge",
   "version": "edge",
+  "Init": true,
```

Comment:
It is already there: https://github.com/zigbee2mqtt/hassio-zigbee2mqtt/blob/109fe6bb73acc7af1c2d6c59feac11a63119c71b/zigbee2mqtt-edge/config.json#L20

Reply:
sorry, didnt find that
but it has to be true not false

Reply:
@mdegat01 can you confirm this has to be `true` and not false as you commited in https://github.com/zigbee2mqtt/hassio-zigbee2mqtt/commit/62f74aade44e24718ffbeb119d1fe110b6c9c918 ?

Reply:
for me it works